src/Views/Players/Player.js

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { fetchPlayerById } from '../../services/fetchPlayers';
+
+export default function Player() {
+  const [player, setPlayer] = useState({ teams: {} });
+  const { id } = useParams();
+
+  useEffect(() => {
+    const getPlayer = async () => {
+      const data = await fetchPlayerById(id);
+      setPlayer(data);
+    };
+    getPlayer();
+  }, [id]);
+  return (
```

Comment:
nice work on making the fetch function dependent on a user clicking on a player.